shopinvader_auth_jwt/components/service_context_provider.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 ACSONE SA/NV
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import logging
+
+from odoo import _
+from odoo.exceptions import MissingError
+
+from odoo.addons.component.core import Component
+
+_logger = logging.getLogger(__name__)
+
+
+class AuthApiKeyServiceContextProvider(Component):
+    _name = "auth_jwt.shopinvader.service.context.provider"
+    _inherit = "shopinvader.service.context.provider"
+    _usage = "auth_jwt_component_context_provider"
+
+    def _get_partner(self):
```

Comment:
```suggestion
    def _get_shopinvader_partner(self):
```

To avoid confusion, since this returns a shopinvader.partner ?